apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/dgs/DGSObserverTest.java

```diff
@@ -76,7 +78,15 @@
             DGSServiceImpl.class,
             DGSObserver.class,
             DerivedDbTablesRegistryImpl.class,
-            EpochTime.class, BlockDaoImpl.class, TransactionDaoImpl.class)
+            EpochTime.class, BlockDaoImpl.class, TransactionDaoImpl.class,
```

Comment:
Do not add all CDI classes, just mock required

Reply:
partially done, this test depends on the account service/table behavior.